app/data/2017/speakers.js

```diff
@@ -0,0 +1,479 @@
+      message: '',
+      title: 'From Localhost to Production',
+    },
+    twitter: 'https://twitter.com/gautam',
+    website: 'http://www.gautamarora.com/',
+    workshop: null,
+  },
+  {
+    company: 'SEEK',
+    crazy: '',
+    description:
+      "<p>Mark Dalgleish is the co-creator of CSS Modules, lead organiser of MelbJS, and DesignOps Lead at SEEK, Australia's leading job site. Having got his start with HTML and UI design at a young age, Mark has since developed a love of open source and software engineering but always as a means to creating elegant, usable experiences.</p>",
+    fullName: 'Mark Dalgleish',
+    github: 'https://github.com/markdalgleish',
+    img: 'mark.jpg',
+    name: 'mark',
+    otherTalks: [],
+    permalink: 'mark-dalgleish',
+    position: 'DesignOps Lead',
+    previousTalks: [],
+    reference: 'him',
+    talk: {
+      description:
+        "In the past few years, we've witnessed a massive increase in the amount of CSS experimentation, with ideas like CSS Modules and—most controversially—the rise of CSS-in-JS. But does mixing our styles and logic run counter to the original ideas of CSS? Does it break progressive enhancement? Most importantly, when dealing with such a boundary-defying trend, how can we ensure that the CSS and JS communities are united rather than divided?",
+      message: '',
+      title: 'A Unified Styling Language',
+    },
+    twitter: 'https://twitter.com/markdalgleish',
+    website: '',
+    workshop: null,
+  },
+  {
+    company: 'Evozon',
+    crazy: '',
+    description:
+      '<p>Developer, writer, educator and father, on a mission to contribute to a future where technology serves humanity - not the other way around. Expert diaper-changer with a ridiculous passion for Open Source.</p>',
+    fullName: 'Dan Călinescu',
+    github: 'https://github.com/idancali',
+    img: 'calinescu.jpg',
+    name: 'calinescu',
+    otherTalks: [],
+    permalink: 'dan-calinescu',
+    position: 'Knowledge Manager',
+    previousTalks: [],
+    reference: 'him',
+    talk: {
+      description:
+        'Chunky is an opinionated framework that helps you write better React Apps by organizing functionality in modular, cross-functional chunks.',
+      message: '',
+      title: 'Introducing Chunky - The Full Stack React Framework',
+    },
+    twitter: 'https://twitter.com/idancali',
+    website: 'https://medium.com/@idancali',
+    workshop: null,
+  },
+  {
```

Comment:
william didn't join in the end, you can remove him

Reply:
done